### ISSUE TITLE
bug: full screen color is broken

### ISSUE THREAD

**Issue by qwmnerbvqwmn** [crdroidandroid/issue_tracker#598]
### Issue type

User interface

### Device

POCO F2 Pro

### crDroid version

crDroid 10

### Exact version / Build date

10.11/2024-12-17

### Bug description

in some app, the full screen color is broken, grey color will displayed red

### Steps to reproduce

Step 1.  enable "auto-rotate screen", and set the screen brightness to lowest.
Step 2.  launch the telegram app, send a black color with gradient to anyone. this picture is a great example: 
https://thumbs.dreamstime.com/b/simple-black-white-gradient-background-image-can-be-used-presentations-websites-other-design-projects-338855640.jpg
Step 3. rotate the phone, so it will trigger android rotate screen.
now the screen color will displayed broken.

above is reproduce steps in telegram app, in tencent tim 4.0.98, when you set the screen brightness to lowest, directly enter the app login account, the bug is directly displayed.

### Relevant log (logcat/build log)

```shell
log is more than 64KB, so attach it in Screenshots or videos
```

### Screenshots or videos

log.log
[log.log](https://github.com/user-attachments/files/18280682/log.log)

this is broken screen display.
![IMG20241231163915](https://github.com/user-attachments/assets/07c07db7-d4e5-4c10-a915-fc7979eddc78)

this is normal screen display.
![IMG20241231163950](https://github.com/user-attachments/assets/6a6636c1-892c-4117-90da-982d20b1fdbb)

tencent tim broken screen display.
![IMG20241231174106](https://github.com/user-attachments/assets/058fc016-51d4-4c9d-a277-ad55aa8ae8e0)

system quick setting broken screen display. this appear after launch tencent tim
![IMG20241231174054](https://github.com/user-attachments/assets/77c45c59-28df-4303-a0ea-decb7a7d7300)

system quick setting normal screen display
![IMG20241231174117](https://github.com/user-attachments/assets/c255843c-e43e-4dbb-8ed6-c5f4a8f60f1b)

### Acknowledgements

- [X] I've checked device is officially supported (for device specific reports and not source related).
- [X] I'm running latest version available on crdroid.net for this device
- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by jason-eu** [comment]
Can you turn off 77hz and try to reproduce the issue please.

**Comment by qwmnerbvqwmn** [comment]
> Can you turn off 77hz and try to reproduce the issue please.

yes, 77hz is the reason for screen color broken, after turn off 77hz, the screen no longer be broken red

**Comment by qwmnerbvqwmn** [comment]
after enable fps, The reason is clearer now

![Image](https://github.com/user-attachments/assets/0169929f-b9cd-4426-a243-6d58a0a17fae)

![Image](https://github.com/user-attachments/assets/d2f7f89b-ad76-47bb-bf29-1c5ab586f464)

and this is the 60hz / 77hz / 60~77hz switch button

![Image](https://github.com/user-attachments/assets/375880eb-d0de-4ba3-bcb7-06eb537861db)

![Image](https://github.com/user-attachments/assets/81d6ec72-52f4-4b35-a162-aceb718b2ebc)

**Comment by jason-eu** [comment]
Well done.

Be sure to join the "crdroidlmi" group on Telegram.

You should also be able to find the 75hz dtbo there that someone used on Android 14 and didn't have any flashbang but I don't remember if they had black crush or not.

Ask the guy with the "GATEKEEPER" flair.

**Comment by qwmnerbvqwmn** [comment]
> Well done.
> 
> Be sure to join the "crdroidlmi" group on Telegram.
> 
> You should also be able to find the 75hz dtbo there that someone used on Android 14 and didn't have any flashbang but I don't remember if they had black crush or not.
> 
> Ask the guy with the "GATEKEEPER" flair.

ok, i joined the crdroidlmi group a few seconds ago, thank you!
